runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -80,6 +80,8 @@ MM_StandardAccessBarrier::initialize(MM_EnvironmentBase *env)
 	if (!_generationalAccessBarrierComponent.initialize(env)) {
 		return false;
 	}
+	_scavenger = MM_GCExtensions::getExtensions(env)->scavenger;
+	Assert_MM_true(NULL != _scavenger);
```

Comment:
It's actually possible to be null. An obvious config is Optavgpause, but I think even Optthruput will create SAB (although barely use it, not so much for real barrier stuff, but for some java specific API)